src/components/Nav.js

```diff
@@ -1,16 +1,32 @@
-import { Link } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
+import { FaRegUserCircle } from 'react-icons/fa';
 
 const NavBar = () => (
   <nav className="Nav-parent-container">
     <h1>Bookstore CMS</h1>
     <ul className="nav-container">
       <li>
-        <Link to="/"> Books </Link>
+        <NavLink
+          to="/"
+          className={({ isActive }) => (isActive ? 'link-active' : 'link')}
+        >
+          {' '}
+          BOOKS
+          {' '}
+        </NavLink>
       </li>
       <li>
-        <Link to="/categories">Categories</Link>
+        <NavLink
+          to="/categories"
+          className={({ isActive }) => (isActive ? 'link-active' : 'link')}
+        >
+          CATEGORIES
+        </NavLink>
       </li>
     </ul>
+    <div className="user_icon">
+      <FaRegUserCircle className="fa-icon" />
+    </div>
```

Comment:
- [ ] I noticed that the user icon is not aligned to the books 😓 Please make sure to align it as given in the Zeplin guidelines. See the images below for more clarity 💡

Your design 🔽 | Zeplin design 🔽
---| ---|
![image](https://user-images.githubusercontent.com/106119105/231280846-f66d3cd3-19ad-453a-8db0-00288f01ed4c.png) | ![image](https://user-images.githubusercontent.com/106119105/231281242-8cf6709e-cbd2-40e8-8c9a-2f4b8457ff95.png)